presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionSuite.scala

```diff
@@ -25,7 +25,7 @@ class CompletionSuite extends BaseCompletionSuite:
         |  Lis@@
         |}""".stripMargin,
       """
-        |List scala.collection.immutable
+        |List:  scala.collection.immutable
```

Comment:
@rochala the `List` accessor changed from a getter into a proper value. I see that this has changed. Is this the correct output? Or is this a symptom of some logic not identifying the accessor correctly and not dealiasing it? I remember that there are some special cases for these aliases.

Reply:
Yes, this is the correct completion label.

Reply:
Is this PR going to be backported to LTS or is it targeted for 3.4 ?

Reply:
Not sure if we can. It would break some macros. We should at least see what happens in the open community build before we consider it as an option.

Reply:
OK, this will make backporting a bit harder as tests will diverge between LTS and Next. This is not a big deal, but will be a small pain. I'll investigate whether the labels can print in the same way.